README.md

```diff
@@ -0,0 +1,34 @@
+Understanding the Contents
+---------------------------
+
+This repository contains a simple *hello /* web application that runs on your *localhost* on port number *8080*.
+
+The application is written in *Golang* and the code for the same can be found in the file *main.go*.
+
+Another file that can be found is *Dockerfile* that is used to compile and run the application in a *Docker container*.
+***
+
+Installing and Starting the Application
+----------------------------------------
+
+To run the application, follow the following instructions: -.
+
+After the repository is cloned in your local machine, make sure you are present in the same directory where the repository is cloned.
+
+Type in the command '"docker build --tag=<Any tag name of your choice>"'.
```

Comment:
You should format code using ``` code example ``` so its easier to read.

Reply:
please read the document i sent you on commenting and markdown practices

Reply:
I thought this is a command, therefore I didn't use the code format. Should I use the code format for commands as well going forward ?

Reply:
And yes, I did read the document

Reply:
yes you should use the code format for commands so it can help engineers differentiate code from instructions